src/main/java/hudson/plugins/selenium/HubLauncher.java

```diff
@@ -37,21 +37,21 @@ public HubLauncher(int port, String[] args, Level logLevel) {
 
     public Void call() {
         try {
-            Logger LOG = Logger.getLogger(HubLauncher.class.getName());
+            Logger log = Logger.getLogger(HubLauncher.class.getName());
```

Comment:
In most places the logger is called LOGGER, here it was called LOG and you're replacing it for the lowercase log, but the uppercase LOGGER is not replaced anywhere. I understand that loggers are usually written in uppercase even though they are not statics, could you tell me why you chose to replace LOG with log instead of LOGGER, or why you replaced it here but not elsewhere?